src/test/serialization_tests.cpp

```diff
@@ -82,6 +83,46 @@ const std::vector<ovms::Precision> UNSUPPORTED_OUTPUT_PRECISIONS{
     // TODO: There are new API 2.0 precisions we do not support. Add tests for those.
 };
 
+const std::vector<ovms::Precision> SUPPORTED_KFS_OUTPUT_PRECISIONS{
+    // ovms::Precision::UNDECIFIED,
+    // ovms::Precision::MIXED,
+    ovms::Precision::FP64,
+    ovms::Precision::FP32,
+    ovms::Precision::FP16,
+    // InferenceEngine::Precision::Q78,
+    ovms::Precision::I16,
+    ovms::Precision::U8,
+    ovms::Precision::I8,
+    ovms::Precision::U16,
+    ovms::Precision::I32,
+    ovms::Precision::I64,
+    ovms::Precision::U32,
+    ovms::Precision::U64,
+    // ovms::Precision::BIN,
+    ovms::Precision::BOOL
+    // ovms::Precision::CUSTOM)
+};
+
+const std::vector<ovms::Precision> UNSUPPORTED_KFS_OUTPUT_PRECISIONS{
+    // ovms::Precision::UNDEFINED, // Cannot create tensor with such precision
+    // ovms::Precision::MIXED, // Cannot create tensor with such precision
+    // ovms::Precision::FP64,
+    // ovms::Precision::FP32,
+    // ovms::Precision::FP16,
+    // ovms::Precision::Q78, // Cannot create tensor with such precision
+    // ovms::Precision::I16,
+    // ovms::Precision::U8,
+    // ovms::Precision::I8,
+    // ovms::Precision::U16,
+    // ovms::Precision::I32,
+    // ovms::Precision::I64,
```

Comment:
This is already defined in test_utils.hpp

Reply:
But this one has different content than in test_utils (commented precisions due to creating ov:tensor in tests).